docs/Set-PSReadLineKeyHandler.md

```diff
@@ -1,150 +1,157 @@
 ---
+external help file: Microsoft.PowerShell.PSReadLine.dll-Help.xml
+keywords: powershell,cmdlet
+locale: en-us
+Module Name: PSReadLine
+ms.date: 12/07/2018
+online version: http://go.microsoft.com/fwlink/?LinkID=821452
 schema: 2.0.0
-external help file: Microsoft.PowerShell.PSReadLine2.dll-help.xml
+title: Set-PSReadLineKeyHandler
 ---
 
 # Set-PSReadLineKeyHandler
 
 ## SYNOPSIS
-
-Binds or rebinds keys to user defined or PSReadLine provided key handlers.
+Binds keys to user-defined or PSReadLine key handler functions.
 
 ## SYNTAX
 
-### Set 1
+### ScriptBlock
 
 ```
-Set-PSReadLineKeyHandler [-Chord] <String[]> [-ScriptBlock] <ScriptBlock> [-BriefDescription <String>]
- [-Description <String>] [-ViMode <ViMode>]
+Set-PSReadLineKeyHandler [-ScriptBlock] <ScriptBlock> [-BriefDescription <String>] [-Description <String>]
```

Comment:
Why are the parameters reordered here? This doesn't match the `Get-Command -Syntax` output which I think is better than this change.

Reply:
Not sure what happened here. Will fix this.